core/src/main/scala/org/apache/spark/sql/execution/ddl.scala

```diff
@@ -75,7 +75,7 @@ private[sql] case class DropTableCommand(ifExists: Boolean,
   override def run(session: SparkSession): Seq[Row] = {
     val snc = session.asInstanceOf[SnappySession]
     val catalog = snc.sessionState.catalog
-    snc.dropTable(catalog.newQualifiedTableName(tableIdent), ifExists, resolveRelation = false)
+    snc.dropTable(catalog.newQualifiedTableName(tableIdent), ifExists, resolveRelation = true)
```

Comment:
Why we needed to change this ?

Reply:
Was missed in previous fix. The "resolveRelation=false" route is only for smart connector mode when the required classes for an external table may not be present on embedded-side so relation should not be attempted to be materialized.